spec/controllers/feedbacks_controller_spec.rb

```diff
@@ -60,13 +60,15 @@ def valid_session
         assigns(:feedback).should be_a_new(Feedback)
         assigns(:user_name).should eq(@user.name)
       end
+    end
+  end
 
-      it "loads the existing feedback if one exists for this user" do
-        feedback = create(:feedback, :review => @review, :user => @user)
-        get :new, {:review_id => @review.id}, valid_session
-        assigns(:feedback).should eq(feedback)
-        assigns(:user_name).should eq(@user.name)
-      end
+  describe "GET new_additional" do
+    it "should create a new external feedback" do
+      feedback = create(:feedback, :review => @review, :user => @user)
```

Comment:
Useless assignment to variable - `feedback`.<br>Use the new Ruby 1.9 hash syntax.